src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -935,25 +938,31 @@ public void testInspectContainerWithSecurityOpts() throws Exception {
 
   @Test
   public void testContainerWithHostConfig() throws Exception {
```

Comment:
@davidxia Yes, I noticed that. So are you guys willing to drop support for 1.15 entirely? I am happy to update the PR to make the API conform to only API version 1.18